docs/v2_migration.md

```diff
@@ -124,7 +124,7 @@ with rasterio.open("my_tif.tif") as src_dst:
 with rasterio.open("my_tif.tif") as src_dst:
     mercator_tile = mercantile.Tile(x=tile_x, y=tile_y, z=tile_z)
     tile_bounds = mercantile.xy_bounds(mercator_tile)
-    t, m = rio_tiler.utils._tile_read(src, tile_bounds, 256, tile_edge_padding=4, minimum_tile_cover=0.3)
+    t, m = rio_tiler.utils._tile_read, tile_bounds, 256, tile_edge_padding=4, minimum_tile_cover=0.3)
```

Comment:
1. This is definitely a typo because now it's invalid syntax
2. Both here and below you're not using `src_dst` at all, so it doesn't make sense
3. I guess this is for v1, but I'd recommend not advertising private functions.

Reply:
🤦